app/src/main/java/com/github/se/icebreakrr/ui/sections/shared/DisplayProfile.kt

```diff
@@ -50,6 +53,21 @@ import com.github.se.icebreakrr.ui.tags.TagStyle
  * @param listOfTags A list of tags related to the user, paired with colors.
  * @param description The user's detailed description.
  */
+val informationTitleSize = 14.sp
+val informationTitleLineHeight = 20.sp
+val informationTitleFontWeight = 500
+val informationTitleLetterSpacing = 0.1.sp
+val catchPhraseSize = 22.sp
+val catchPhraseLineHeight = 28.sp
+val catchPhraseWeight = 400
+val descriptionFontSize = 16.sp
+val descriptionLineHeight = 24.sp
+val descriptionFontWeight = 500
+val descriptionLetterSpacing = 0.15.sp
+val tagHeight = 40
+val requestButtonSize = 55.dp
+val requestButtonElevation = 8.dp
+
```

Comment:
Nice ! All the constants are now stored in the shared folder, that makes hte code clearer and easier to maintain